index.js

```diff
@@ -1,18 +1,50 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
+// added defer to html file script tag in header so that the Javascript loads after html loads.
 
 
 /***** Deliverable 2 *****/
+header.style.color = "#7F6A7C"
 
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+// const travelerDiv = document.querySelector('div.traveler')
+const profileDiv = document.querySelector('div#profile')
 
+const image = profileDiv.querySelector('img')
+image.src = traveler.photo
+image.alt = traveler.name
 
-/***** Deliverable 4 *****/
+travelerName = profileDiv.querySelector('h2')
+travelerName.innerHTML = traveler.name
+
+travelerNickname = profileDiv.querySelector('em')
+travelerNickname.innerHTML = traveler.nickname
```

Comment:
Great! I like the variables to be indicative of their element. So something like:

```
const travelerDiv = document.querySelector('div#profile')

const travelerImg = travelerDiv.querySelector('img')
travelerImg.src = traveler.photo
travelerImg.alt = traveler.name

const travelerNameH2 = travelerDiv.querySelector('h2')
travelerNameH2.textContent = traveler.name 

const travelerNicknameEm = travelerDiv.querySelector('em')
travelerNicknameEm.textContent = traveler.nickname 
```

LOVE that you made sure to add the alt attribute for the image. Sylwia would be proud!